app/components/publishers/vacancies_component/vacancies_component.html.slim

```diff
@@ -0,0 +1,37 @@
+= render(Shared::DashboardComponent.new(heading: heading, navigation_items: vacancy_links, link: organisation_jobs_path))
+
+.vacancies-component
+  .govuk-grid-row.vacancies-component__header class="govuk-!-margin-top-7"
+    .govuk-grid-column-one-third
+      h2.govuk-heading-l class="govuk-!-margin-bottom-4" = t("jobs.#{@selected_type}_jobs_with_count_html", count: @vacancies.count)
+
+    .govuk-grid-column-two-thirds
+      - if @vacancies.any?
+        - if sortable
+          div class="govuk-!-margin-bottom-4"
+            = form_for @filters_form, as: "", url: jobs_with_type_organisation_path(@selected_type), html: { data: { 'auto-submit': true }, class: "sort-inline" }, method: :patch do |f|
+              = f.govuk_collection_select :sort_column,
+                Organisation::JOB_SORTING_OPTIONS[@selected_type],
+                :last,
+                :first,
+                label: { text: t("jobs.sort_by.label"), size: "s", class: "govuk-label inline govuk-!-margin-right-2" }
+
+              = f.govuk_submit t("jobs.sort_by.submit")
+
+      - if @organisation.is_a?(SchoolGroup)
+        button.govuk-button.govuk-button--secondary class="govuk-!-margin-bottom-0 moj-action-bar__filter" id="toggle-filters-sidebar"
+          = t("buttons.hide_filters")
+
+  .govuk-grid-row
+    .govuk-grid-column-full-width
+      .moj-filter-sidebar class="govuk-!-margin-bottom-5"
+        - if @organisation.is_a?(SchoolGroup)
+          .govuk-grid-column-one-third
+            = form_for @filters_form, url: organisation_managed_organisations_path(anchor: "search-results"), html: { method: "patch", data: { "auto-submit": true } } do |f|
```

Comment:
the `method: "patch"` can be outside the html hash and just be `method: :patch` like it is on line 12